README.md

```
@@ -19,6 +19,7 @@
 *   **直感的なUI/UX:**
     *   HPバーやアニメーションなど、視覚的に戦況を把握しやすいデザインを追求しました。
     *   GSAPを活用し、SF的な世界観に合わせた滑らかなアニメーションを実装することで、使う楽しさを演出しました。
+    *   モバイル版 (Beta) でも「チーム合計耐久力予測」と「選択キャラクター詳細」を確認できるようにし、どの端末でも主要な情報にアクセスしやすくしました。
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)